src/components/AudioPlayer/Audio/Audio.component.jsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/default-props-match-prop-types */
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable react/require-default-props */
+import React from 'react';
+import propTypes from 'prop-types';
+
+const AudioComponent = ({ autoPlay, songLink }) => (
+  <div className="">
+    <audio id="" controls autoPlay={autoPlay}>
+      <track kind="captions" />
+      <source src={songLink} type="audio/mpeg" />
+      <source src={songLink} type="audio/wav" />
+      <source src={songLink} type="audio/ogg" />
```

Comment:
Is there a way we can use a single type depending on the actual file?

Reply:
i couldnot find a solution for that

Reply:
Please add a ticket on Jira, let's fix this later.